doc/nrf/ug_thingy91_gsg.rst

```diff
@@ -0,0 +1,571 @@
+   #. Scroll down in the menu on the left to :guilabel:`Device` and click :guilabel:`Erase & write`.
+
+      .. figure:: /images/programmer_ext_debug_hex_write.png
+         :alt: Programming using External debug probe
+
+         Programming using External debug probe
+
+      The update is completed when the animation in Programmer's Device memory layout window ends.
+
+   #. Scroll up in the menu on the left to :guilabel:`File` and click :guilabel:`Clear files`.
+
+   .. programing_ext_dp_end
+
+4. Update the modem firmware on the nRF9160 System in Package (SiP):
+
+   a. Set the SWD selection switch **SW2** to nRF91.
+
+   #. In the menu on the left, click :guilabel:`Add file > Browse...`.
+
+      .. figure:: /images/programmer_add_file.png
+         :alt: Programmer - Add file
+
+         Programmer - Add file
+
+      A file explorer window appears.
+
+   #. Navigate to the folder you downloaded and extracted from `Nordic Semiconductor website`_ in the :ref:`Download firmware <download_firmware_thingy91>` step.
+   #. Find the modem firmware zip file with the name similar to mfw_nrf9160_*.zip and the number of the latest version and click :guilabel:`Open`.
+
+      .. note::
+         Do not extract the modem firmware zip file.
+
+   #. Select the zip file and click :guilabel:`Open`.
+   #. In the Programmer application, scroll down in the menu on the left to :guilabel:`Device` and click :guilabel:`Write`.
+
+      .. figure:: /images/programmer_ext_debug_update_modem.png
+         :alt: Programmer - Update modem
+
+         Programmer - Update modem
+
+      The :guilabel:`Modem DFU` window appears.
+   #. In the :guilabel:`Modem DFU` window, click :guilabel:`Write`.
+
+      .. note::
+         If you have issues updating modem firmware, click :guilabel:`Erase all` before trying to update the modem again. In this case, the contents of the flash memory are deleted and the applications must be reprogrammed.
+
+      When the update is complete, a :guilabel:`Completed successfully` message appears.
+
+5. Update the nRF9160 SiP application:
+
+   a. Make sure the SWD selection switch **SW2** is set to nRF91.
+
+   #. In the menu on the left, click :guilabel:`Add file > Browse...`.
+
+      .. figure:: /images/programmer_add_file.png
+         :alt: Programmer - Add file
+
+         Programmer - Add file
+
+      A file explorer window appears.
+
+   #. Navigate to the folder you downloaded and extracted from `Nordic Semiconductor website`_ in the :ref:`Download firmware <download_firmware_thingy91>` step.
+
+   #. Open the folder that contains the HEX files for updating over USB.
+      See the :file:`CONTENTS.txt` file for information on which file you need.
+
+   #. Select the appropriate Asset Tracker v2 firmware file.
+
+.. include:: ug_thingy91_gsg.rst
+   :start-after: programing_ext_dp_start
+   :end-before: programing_ext_dp_end
+
+..
+
+Next, you need to create an nRF Cloud account if you do not have one already.
```

Comment:
I'm not sure of this is needed. These comments at the ends of topics were added to the HTML version because the Infocenter structure is not as linear as a PDF or NCS page and it's not as obvious what you need to do next.